api/system-backup/read

```diff
@@ -104,6 +104,28 @@ sub find_latest_log
     return ($latest_filename, $latest_ts);
 }
 
+sub find_all_logs
+{
+    my $name = shift;
+    my $n = "[0123456789]";
+    my $logs;
+    foreach my $file (glob("/var/log/backup/backup-$name-$n$n$n$n$n$n$n$n$n$n$n$n.log")) {
+        my $stat = stat($file);
+        if ( -f $file ) {
+            open my $fh, '<', $file;
+            $/ = undef;
+
+            $file =~ s/[^0-9]//g;
+            my ($y,$M,$d,$h,$m) = $file =~ /^([0-9]{4})([0-9]{2})([0-9]{2})([0-9]{2})([0-9]{2})\z/ or die;
```

Comment:
Never die inside an API or it will return a malformed JSON for the UI.
In case of error, use the `error()` function.